sw/device/tests/alert_handler_reverse_ping_in_deep_sleep_test.c

```diff
@@ -156,6 +161,30 @@ static void alert_handler_config(void) {
       &alert_handler, kDifAlertHandlerIrqClassb, kDifToggleEnabled));
 }
 
+/**
+ * Log any external alerts.
+ */
+static void log_external_alerts(void) {
+  for (int i = 0; i < ALERT_HANDLER_PARAM_N_ALERTS; ++i) {
+    bool is_cause;
+    CHECK_DIF_OK(
+        dif_alert_handler_alert_is_cause(&alert_handler, i, &is_cause));
+    if (is_cause) {
+      LOG_INFO("Unexpected alert cause: %d", i);
+
+      switch (i) {
+        case kTopEarlgreyAlertIdFlashCtrlFatalErr: {
+          dif_flash_ctrl_faults_t faults_out;
+          CHECK_DIF_OK(
+              dif_flash_ctrl_get_faults(&flash_ctrl_state, &faults_out));
+          LOG_INFO("Flash ctrl faults: %d", faults_out);
+          break;
+        }
```

Comment:
you are missing a `default` case; see [style guide](https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements)